frontend/src/metabase/account/password/actions.js

```diff
@@ -40,32 +39,3 @@ export const updatePassword = createThunkAction(UPDATE_PASSWORD, function(
     }
   };
 });
-
-export const validatePassword = createThunkAction(VALIDATE_PASSWORD, function(
-  password,
-) {
-  return async function(dispatch, getState) {
-    return await UtilApi.password_check({
-      password: password,
-    });
-  };
-});
-
-export const updateUser = createThunkAction(UPDATE_USER, function(user) {
```

Comment:
I am a little concerned that this function was removed.

Reply:
Me too, but it turned on that they were not used by the underlying components.